src/pages/PictureUploadBox.tsx

```diff
@@ -86,7 +86,16 @@ const PictureUploadBox = ({
       onDragOver={(e) => e.preventDefault()}
       {...getStackProps(props)}
     >
-      <Title size={3}>{t('pictures.title')}</Title>
+      <Inline>
+        <Title
+          size={3}
+          css={`
+            min-height: 26px;
+          `}
```

Comment:
```suggestion
          minHeight={'26px'}
```

This will also work I think

Reply:
True ✅